_posts/2020-08-01-Making-IrishGen-SPARQL-Part-III.md

```diff
@@ -0,0 +1,90 @@
+usefulness of this is that if the user knows the URL, they can
+interrogate the Triplestore about what is stored in the Triplestore
+about that URL without needing to go through the ceremony of a
+construct query.  Moreover, describe queries are often used in visual
+representations of RDF graphs.
+
+For instance, the screenshot below is from the GraphDB's "Visual
+Graph" feature and uses Finn mac Cumaill as the exemplar.  What
+happens is that the user sends the URL in which they are interested
+and the system renders the RDF returned as a visual representation.
+This is very useful when investigating single individuals within
+IrishGen.  Additionally, it is good if a user wants to have a secure,
+single starting point to explore the graph in various ways.
+
+<img src="{{site.baseurl}}/assets/images/describe_part3.png" />
+
+Further describe queries can then be executed to investigate
+connections further and the data returned can be integrated into the
+visual representation of the graph.
+
+# Ask
+
+The ask query has one use to inform the user if a basic graph pattern
+has a solution or not.  No other information than a boolean that
+indicates existence of a solution is returned.
+
+In terms of IrishGen, the use of this query form is to ask for a quick
+indication if the information that a user is looking for exists within
+IrishGen.  This can be useful for what are called [Federated
+Queries](https://www.w3.org/TR/sparql11-federated-query/), which is an
+advanced SPARQL topic which will be covered in a future post.
+Although, in terms of how IrishGen is currently configured, incoming
+Federated Queries are not possible and outgoing Federated Queries are
+of minimal usefulness while Linked Data in Celtic Studies is not
+ubiquitous.
+
+# Conclusion
+
+This concludes the series on SPARQL in terms of IrishGen.  While not
+every possible combination was explored, these posts provide a basis
+for further exploring the IrishGen database.  Query languages like
+SPARQL, most famously [SQL](https://en.wikipedia.org/wiki/SQL), form
+the backbone of data intensive systems.  Learning to manipulate these
+directly will give a user more power than any interface to a database
+could.
+
+From here, the user will need to use their new found SPARQL skills to
+explore IrishGen and other datasets.  For instance, Wikipedia has two
+widely used datasets: [DBPedia](https://wiki.dbpedia.org/) and
+[Wikidata](https://www.wikidata.org/wiki/Wikidata:SPARQL_query_service/Wikidata_Query_Help).
+Practising and using IrishGen will cement the core skills presented in
+this series and give the user a sense of the possibilities for further
+research.
+
+Future posts will cover other aspects that are part of the SPARQL
+standard but not core to the user experience such as [Federated
+Queries](https://www.w3.org/TR/sparql11-federated-query/) and [Named
+Graphs](https://en.wikipedia.org/wiki/Named_graph).  These future
```

Comment:
Named Graphs are how the user can query one particular manuscript version, no? I'd argue that this is core to the user experience, as serious researchers won't be able to use the data without it.